app/forms/mappings/LocalDateFormatter.scala

```diff
@@ -108,6 +107,52 @@ private[mappings] class LocalDateFormatter(
       s"$key.year" -> value.getYear.toString
     )
 
+    /**
+   * Updates the FormError key as per valid day, month and year criteria
+   * key is updated as below
+   * invalid day - key.day
```

Comment:
Could you please update the comments as we don't have day here? So day reference must be removed